src/utils/database/weeklyquestion_table.py

```diff
@@ -0,0 +1,72 @@
+"""
+Weekly Question table module
+"""
+
+from typing import List
+from tinydb import TinyDB, where
+from tinydb.table import Document
+from .db_decorators import validate_insert
+
+class WeeklyQuestionTable:
+    """
+    A class to act on the weeklyquestion_table of the database
+    """
+    TABLE_WEEKLY_QUESTION_FIELDS = [
+        "id",
+        "challenge_id",
+        "title",
+        "title_slug",
+        "difficulty",
+    ]
+    TABLE_WEEKLY_QUESTION = "Weekly_Question"
```

Comment:
move to __init__() as self.table_fields and self.table_name

Reply:
Renamed but kept outside of constructor due to issues with the validate_insert decorator.